examples/aggregator/gcp/pod.yaml

```diff
@@ -24,6 +27,26 @@ spec:
       env:
         - name: RUST_LOG
           value: info
+    - name: prometheus-to-sd
+      image: gcr.io/google-containers/prometheus-to-sd:v0.9.0
+      ports:
+        - name: profiler
+          containerPort: 6060
+      command:
+        - /monitor
+        - --stackdriver-prefix=custom.googleapis.com
```

Comment:
Does this need to be this exact value? If so, could you add a link to some source?